broker/plugins/kubejobs/plugin.py

```diff
@@ -13,53 +13,83 @@
 # limitations under the License.
 
 import datetime
+import json
```

Comment:
There should be a line between json and redis import (https://legacy.python.org/dev/peps/pep-0008/#imports)